config.rb

```diff
@@ -121,4 +121,5 @@ def self.test?
   optional :ubicloud_images_blob_storage_endpoint, string
   optional :ubicloud_images_blob_storage_access_key, string, clear: true
   optional :ubicloud_images_blob_storage_secret_key, string, clear: true
+  optional :ubicloud_images_blob_storage_certs, string
```

Comment:
oof, another config option that doesn't seem to have a streamlined development default so one can dip into the system? we need to do something about these, it's not unique to this patch though.

Reply:
Since our root certificate is public, can we hardcode it?

Reply:
I think we're multi-root, right? As in, we prolifically generate roots? I suppose we could roll in a development cert, but alternatively, we can have a special "development" minio ID to look up and grab that root.

Reply:
like, although I have a bit of misgiving about dev_project, I think it's kinda an okay general idea

Reply:
This was not necessary for production but development. We wanted to be able to test against different minio clusters from dev. Such as, downloading an image that exists in the production environment.